addons/binding/org.openhab.binding.neato/src/main/java/org/openhab/binding/neato/internal/classes/BeehiveAuthentication.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.neato.internal.classes;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link BeehiveAuthenticcation} is the internal class for handling authentication.
+ *
+ * @author Patrik Wimnell - Initial contribution
+ */
+public class BeehiveAuthentication {
+
+    @SerializedName("access_token")
```

Comment:
These `@SerializedName` should stay (because the mapped name is different), but afaik the `@Expose` not.